src/components/editions/headerImage.tsx

```diff
@@ -77,6 +112,15 @@ const sizes: Sizes = {
 	default: '100vw',
 };
 
+const fullWidthSizes: Sizes = {
+	mediaQueries: [],
```

Comment:
Should we be using this `mediaQueries` input to add dynamic image sizes for the different breakpoints? Even if it's just tablet and mobile.

Reply:
It's full width across all devices for Interview so I don't think there would be break point sizes?

Reply:
My understanding is that the `Img` component pulls in a series of sources for the `<picture />` element based on the values we provide to `mediaQueries`.

So for example when we give it an input of:

```
	mediaQueries: [
		{ breakpoint: 'tablet', size: '740px' },
		{ breakpoint: 'wide', size: '980px' },
	],
```

The browser will use the picture element to automatically load in the appropriate sized image for that breakpoint. If we only provide a `default` value we don't get that.

Appreciate that given this is a template it's only loaded once and therefore we might not get the benefits of that but just seems in keeping with the way the `Img` component is intended to be used.

Reply:
TLDR: the breakpoint sizes relate to the size of image we're loading within the <picture /> element rather than anything to do with layout.

Reply:
Gotcha - will update!

Reply:
Gotcha - will update!